docs/msbuild/common-msbuild-project-items.md

```diff
@@ -125,6 +125,17 @@ In [!INCLUDE[vstecmsbuild](../extensibility/internals/includes/vstecmsbuild_md.m
 | Visible | Optional boolean. Indicates whether to display the file in **Solution Explorer** in [!INCLUDE[vsprvs](../code-quality/includes/vsprvs_md.md)]. |
 | CopyToOutputDirectory | Optional string. Determines whether to copy the file to the output directory. Values are:<br /><br /> 1.  Never<br />2.  Always<br />3.  PreserveNewest |
 
+### InternalsVisibleTo
+ Specifies assemblies to be emitted as `[InternalsVisibleTo(..)]` assembly attributes.
+
+| Item metadata name | Description |
+|-----------------------| - |
+| Include | The assembly name. |
+| Key | Optional string. The public key of the assembly. |
+
+> [!NOTE] 
+> This applies to SDK-style projects only.
```

Comment:
Is this the right phrasing? It applies to projects using the .NET Core SDK only, which is slightly distinct from SDK-style projects, but maybe that's too fine a distinction to draw?